src/twisted/test/test_log.py

```diff
@@ -329,14 +345,7 @@ def setUp(self):
         self.flo = log.FileLogObserver(self.out)
         self.lp.addObserver(self.flo.emit)
 
-        try:
-            "\N{VULGAR FRACTION ONE HALF}"
-        except UnicodeEncodeError:
-            pass
-        # This is the behavior we want - don't change anything.
```

Comment:
I guess this was valid only for py2?

I am not sure why the previous code was needed